docs/wcs/index.rst

```diff
@@ -28,79 +42,103 @@ It performs three separate classes of WCS transformations:
 - table lookup distortions as defined in the FITS WCS `distortion
   paper`_.
 
-Each of these transformations can be used independently or together in
-a standard pipeline.
+Pixel Conventions and Definitions
+---------------------------------
 
-Getting Started
-===============
+Both APIs assume that integer pixel values fall at the center of pixels (as assumed in
+the `FITS WCS standard`_, see Section 2.1.4 of `Greisen et al., 2002,
+A&A 446, 747 <https://doi.org/10.1051/0004-6361:20053818>`_).
 
-The basic workflow is as follows:
+However, there’s a difference in what is considered to be the first pixel. The
+``High Level API`` follows the Python and C convention that the first pixel is
+the 0-th one, i.e. the first pixel spans pixel values -0.5 to + 0.5. The
+``Low Level API`` takes an additional ``origin`` argument with values of 0 or 1
+indicating whether the input arrays are 0- or 1-based.
+The Low-level interface assumes Cartesian order (x, y) of the input coordinates,
+however the Common Interface for World Coordinate System accepts both conventions.
+The order of the pixel coordinates ((x, y) vs (row, column)) in the Common API
+depends on the method or property used, and this can normally be determined from
+the property or method name. Properties and methods containing “pixel” assume (x, y)
+ordering, while properties and methods containing “array” assume (row, column) ordering.
 
-    1. ``from astropy import wcs``
+Some Simple Examples
+====================
 
-    2. Call the `~astropy.wcs.WCS` constructor with an
-       `astropy.io.fits` `~astropy.io.fits.Header` and/or
-       `~astropy.io.fits.HDUList` object.
+One example of the use of the high-level WCS API is to use the
+`~astropy.wcs.wcs.WCS.pixel_to_world` to yield the simplest WCS
+with default values, converting from pixel to world coordinates::
 
-    3. Optionally, if the FITS file uses any deprecated or
-       non-standard features, you may need to call one of the
-       `~astropy.wcs.wcs.WCS.fix` methods on the object.
+    >>> from astropy.wcs import WCS
+    >>> w = WCS('image.fits')  # doctest: +IGNORE_WARNINGS
+    >>> lon, lat = w.pixel_to_world(30, 40)
+    >>> print(lon, lat)
+    31.0 41.0
 
-    4. Use one of the following transformation methods:
+Similarly, another use of the high-level API is to use the
+`~astropy.wcs.wcs.WCS.world_to_pixel` to yield another simple WCS, while
+converting from world to pixel coordinates::
```

Comment:
This method transforms world to pixel coordinates .

Rewrite the example below as 

>>> x, y = w.pixel_to_world(sky)
>>> print(x, y)